### PR TITLE
Update Tag Mod Email

### LINKED ISSUE #19328: Update "Tag Mod" email
**Is your feature request related to a problem? Please describe.**

As a Forem Admin, I want to update the email that new tag moderators receive.

**Describe the solution you'd like**

Update the copy as found in [this GDoc](https://docs.google.com/document/d/1XDzPQhnCBoMbiTrr0ue3VEHd115WBq4NXK3bQVdWHTc/edit?usp=sharing).  _Note: visible to Forem Core Team only_.  (As a result, I'm reserving this issue for Forem Team only).

**Additional context**

Relevant files:
* `tag_moderator_confirmation_email` files in [`/forem/tree/main/app/views/mailers/notify_mailer`](https://github.com/forem/forem/tree/main/app/views/mailers/notify_mailer)

Remember that this will apply to _all Forems_, so we can't link to anything that is `dev.to` specific.  Instead, we can link to the `/community-moderation` page.

As outlined in the GDoc, there is one section (at the very end) that is DEV specific.  We can only include this if we have logic that ensures that the section is included only if it's on the DEV instance.

### PR DESCRIPTION
**Pull request by lboogie04**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Update the email that new tag moderators receive as per the copy [here](https://docs.google.com/document/d/1XDzPQhnCBoMbiTrr0ue3VEHd115WBq4NXK3bQVdWHTc/edit?usp=sharing).

## Related Tickets & Documents

- Related Issue #https://github.com/forem/forem/issues/19328
- Closes #19328 

## QA Instructions, Screenshots, Recordings

Go to the [mailer preview and view the email](http://localhost:3000/rails/mailers/notify_mailer/tag_moderator_confirmation_email).

Screenshot 2023-04-14 at 12 25 30

### UI accessibility concerns?

-

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: contents of the mail were changed.
- [ ] I need help with writing tests

## [optional] Are there any post deployment tasks we need to perform?
No 

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb [hunk 1]
```diff
@@ -1,37 +1,29 @@
...
+<p>Tag moderation is something we're continuously iterating on, so you can expect adjustments and expanding features over time as we receive feedback.</p>
 
-<p>
-  Thanks so much for volunteering your time to benefit <%= community_name %>. We sincerely appreciate your help!.</p>
-<p>
+<p>Of course, if you have any questions, please hit “reply” to this email.</p>
+
+<p>Thanks so much for volunteering your time to benefit <%= community_name %>. Your participation will assist us in better organizing the site via tags, and help us to foster a warm, welcoming, inclusive, spam-free-as-can-be platform.</p>
+
+<% if ForemInstance.dev_to? || ForemInstance.local? %>
```
Comment (Ridhwana) [inline app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb:24]: We can only display this for DEV

File: app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb [hunk 2]
```diff
@@ -1,37 +1,29 @@
...
-  </li>
-  <li>
-    Add #<%= @tag %> to posts if applicable.
-  </li>
-  <li>
-    Update the #<%= @tag %> tag landing page, including submission guidelines and design visible in the sidebar.
-  </li>
+  <li>Remove #<%= @tag %> if it’s been added to an irrelevant post. Only remove tags when they do not fit well for the subject matter of the post. Do add more relevant tags if it will elevate the visibility of the post or make it discoverable to other audiences.</li>
+  <li>Add #<%= @tag %> to posts if applicable. This helps the best content become more easily discoverable!</li>
+  <li>Update the #<%= @tag %> tag landing page, including submission guidelines and design visible in the sidebar.</li>
```
Comment (jaw6) [inline app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb:10]: I find it surprising that these lines are just `@tag` when line 3 is `@tag.name` — I would have expected them to have similar semantics, but maybe it works anyway?

File: app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb (hunk 1 above)
Comment (jaw6) [reply app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb:24]: TIL!

**Review by jaw6 — APPROVED** [review]
Looks good to me :+1:

**Review by rt4914 — APPROVED** [review]

File: app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb (hunk 2 above)
Comment (Ridhwana) [reply app/views/mailers/notify_mailer/tag_moderator_confirmation_email.html.erb:10]: This is a VERY good point, I just doubled checked this and in the view `@tag.name` and `@tag` both output the tag name. However, @tag in the method is the actual tag record. 

```
    @user = params[:user]
    @tag = params[:tag]
    @channel_slug = params[:channel_slug]

    subject = I18n.t("mailers.notify_mailer.moderator", tag_name: @tag.name)
    mail(to: @user.email, subject: subject)
  end
```

🧐 going to sleuth a bit more...

**Pull request merged**